h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -144,6 +146,15 @@ else if( Schemes.HDFS.equals(iceRoot.getScheme()) ) {
     } catch (Throwable ignore) {
       Log.info("S3 subsystem not available");
     }
+
+    try {
+      Class klass = Class.forName("water.persist.PersistGcs");
```

Comment:
There is better way - via SPI.

Reply:
This is just a copy&paste of our previous code. See few lines up. I will change the whole thing to ServiceLoader ?

Reply:
This makes sense for now. We can refactor it later. I don't think it should be done in this PR